src/Commands/ClearSettings.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Commands;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+
+class ClearSettings extends BaseCommand
+{
+    protected $group       = 'Housekeeping';
+    protected $name        = 'settings:clear';
+    protected $description = 'Clears all settings from the database.';
+
+    public function run(array $params)
+    {
+        if (! CLI::prompt('This will delete all settings from the database. Are you sure you want to continue?', ['y', 'n'], 'required') === 'y') {
```

Comment:
```suggestion
        if (CLI::prompt('This will delete all settings from the database. Are you sure you want to continue?', ['y', 'n'], 'required') === 'y') {
```

Reply:
We should also call the: `service('settings')->flush()` because now this command does nothing.

Reply:
Oh wow, I must have started working on this command months ago. I didn't look closely enough at my files last night I guess. Heat went out in my office and I distracted trying to keep it warm enough in here it didn't hurt any of my instruments. 

I guess the command is still a good thing to have so I'll get that cleaned up tonight.

Reply:
Done